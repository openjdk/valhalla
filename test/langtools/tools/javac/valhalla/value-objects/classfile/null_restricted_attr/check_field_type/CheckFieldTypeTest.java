/* @test /nodynamiccopyright/
 * @summary [lw5] check that there can only be one NullRestricted attribute
 * @modules jdk.compiler/com.sun.tools.javac.api
 *          jdk.compiler/com.sun.tools.javac.main
 *          jdk.compiler/com.sun.tools.javac.util
 * @library /tools/lib
 * @compile NullRestrictedOnPrimitive.jcod
 * @compile/fail/ref=CheckFieldTypeTest.out -XDrawDiagnostics CheckFieldTypeTest.java
 */

public class CheckFieldTypeTest {
    void m() {
        NullRestrictedOnPrimitive v = new NullRestrictedOnPrimitive();
    }
}
