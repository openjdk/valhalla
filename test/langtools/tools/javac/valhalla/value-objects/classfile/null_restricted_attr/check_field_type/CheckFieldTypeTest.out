CheckFieldTypeTest.java:13:9: compiler.err.cant.access: NullRestrictedOnPrimitive, (compiler.misc.bad.class.file.header: NullRestrictedOnPrimitive.class, (compiler.misc.attribute.not.applicable.to.field.type: NullRestricted, int))
1 error
